sub.go

```diff
@@ -42,7 +42,7 @@ type SubscriptionParams struct {
 	Coupon                      *string                    `form:"coupon"`
 	Customer                    *string                    `form:"customer"`
 	DaysUntilDue                *int64                     `form:"days_until_due"`
-	Items                       []*SubscriptionItemsParams `form:"items,indexed"`
+	Items                       []*SubscriptionItemsParams `form:"items"`
```

Comment:
Minor but is it worth making sure all those arrays have an explicit test for the multiple element one? I checked and for example the subscription test only adds one item in the `items` hash.

Reply:
Yeah, can't hurt. I added multi-item tests for all of order, order return, plans, and subscriptions.